Signal/src/ViewControllers/AddToBlockListViewController.m

```diff
@@ -31,27 +38,28 @@ @interface AddToBlockListViewController () <CountryCodeViewControllerDelegate, U
 
 @property (nonatomic) UIButton *blockButton;
 
+@property (nonatomic) UITableView *contactsTableView;
+
 @property (nonatomic) NSString *callingCode;
 
+@property (nonatomic, readonly) OWSContactsManager *contactsManager;
+@property (nonatomic) NSArray<Contact *> *contacts;
```

Comment:
The work to improve thread safety around updates is already done in https://github.com/WhisperSystems/Signal-iOS/pull/1928